courses/machine_learning/deepdive2/structured/solutions/3b_bqml_linear_transform_babyweight.ipynb

```diff
@@ -0,0 +1,1162 @@
+{
```

Comment:
<ul><li>"Verify that" -&gt; "Run the following cells to verify that"</li><li>The "<a href="https://github.com/GoogleCloudPlatform/training-data-analyst/raw/3f8f177875ebf0d3e70c0c4d8affcb1dbf09275e/courses/machine_learning/deepdive2/structured/solutions/2_prepare_babyweight.ipynb" target="_blank">2_prepare_babyweight</a>" is broken</li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/training-data-analyst/pull/669/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Reply:
<p>Done</p>